### ISSUE TITLE
I meet the same issue.

### ISSUE THREAD

**Issue by LemonITCN** [etcd-io/etcd#11634]
I meet the same issue.

I use the latest version protobuf in  https://github.com/protocolbuffers/protobuf/releases

and I go get -u github.com/golang/protobuf/protoc-gen-go

then if I modified to google.golang.org/grpc v1.26.0

protobuf build pb error undefined: grpc.SupportPackageIsVersion6.

if I modified to google.golang.org/grpc v1.27.0

then etcd will report err:

_Originally posted by @hesongkang in https://github.com/etcd-io/etcd/issues/11563#issuecomment-587299948_

**Comment by LemonITCN** [comment]
I use protoc-gen-go v1.3.2 to solve this problem temporarily, please author to upgrade grpc version to 1.27. * As soon as possible, thank you.

**Comment by eddycjy** [comment]
Awaiting merge, follow https://github.com/etcd-io/etcd/pull/11564.

**Comment by VineethReddy02** [comment]
Any fix for this?

I get etcd error 

```
# github.com/sercand/kuberesolver
../../pkg/mod/github.com/sercand/kuberesolver@v2.1.0+incompatible/builder.go:108:82: undefined: resolver.BuildOption
../../pkg/mod/github.com/sercand/kuberesolver@v2.1.0+incompatible/builder.go:163:32: undefined: resolver.ResolveNowOption
# go.etcd.io/etcd/clientv3/balancer/picker
../../pkg/mod/go.etcd.io/etcd@v0.0.0-20190709142735-eb7dd97135a5/clientv3/balancer/picker/err.go:32:43: undefined: balancer.PickOptions
../../pkg/mod/go.etcd.io/etcd@v0.0.0-20190709142735-eb7dd97135a5/clientv3/balancer/picker/roundrobin_balanced.go:54:54: undefined: balancer.PickOptions
# go.etcd.io/etcd/clientv3/balancer/resolver/endpoint
../../pkg/mod/go.etcd.io/etcd@v0.0.0-20190709142735-eb7dd97135a5/clientv3/balancer/resolver/endpoint/endpoint.go:112:78: undefined: resolver.BuildOption
../../pkg/mod/go.etcd.io/etcd@v0.0.0-20190709142735-eb7dd97135a5/clientv3/balancer/resolver/endpoint/endpoint.go:180:31: undefined: resolver.ResolveNowOption

```

If I downgrade grpc I get other errors from grpc.

**Comment by shreesha21** [comment]
@eddycjy I see that #11564 got merged recently.  However, it's only available in v3.5.0 and later?
(https://github.com/etcd-io/etcd/pull/11637)
